DigitalLearningSolutions.Web/Styles/trackingSystem/previewCertificate.scss

```diff
@@ -0,0 +1,168 @@
+﻿body, html {
+  background-color: #D9ECFF !important;
+}
+
+.style {
+  color: #000000;
+  font-family: Verdana, Arial, Helvetica, sans-serif !important;
+
+  &1 {
+    @extend .style;
+    font-size: 30px
+  }
+
+  &2 {
+    @extend .style1;

```

Comment:
Is this supposed to be `style1` or `style`?

Reply:
style1, I want it to be 30px and bold